fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section82.java

```diff
@@ -0,0 +1,162 @@
+            }
+            case SCORE_1_2 -> {
+                if (!diceRollHelper.isBetweenScore(1, 2)) {
+                    throw new FabledBusinessException("invalid click: 1 and 2");
+                }
+                // TODO minus 1 for combat
+                playerSession.update(playerDto -> {
+                    var newList = playerDto.withBaseStats(playerDto.baseStats());
+                    var newPlayer = newList.baseStats().withCombat(playerDto.baseStats().combat());
+                    return newList;
+                });
+                addPlayerClickId(ClickOptions.SCORE_1_2.ordinal());
+            }
+            case SCORE_3_4 -> {
+                if (!diceRollHelper.isBetweenScore(3, 4)) {
+                    throw new FabledBusinessException("invalid click: 3 and 4");
+                }
+                addPlayerClickId(ClickOptions.SCORE_3_4.ordinal());
+            }
+            case SCORE_5_6 -> {
+                if (!diceRollHelper.isBetweenScore(5, 6)) {
+                    throw new FabledBusinessException("invalid click: 5 and 6");
+                }
+
+                playerSession.update(playerDto -> {
+                    var newList = playerDto.withPossessions(playerDto.possessions());
+                    // TODO add new possession
+                    // cannot be done yet -> list is immutable!!
+                    // newList.possessions().add("fish");
+                    return newList;
+                });
+                addPlayerClickId(ClickOptions.SCORE_5_6.ordinal());
+            }
+            case FOLLOW_NORTH -> {
+                if (!hasDiceOptionBeenSelected()) {
+                    throw new FabledBusinessException("The Dice Option was not clicked");
+                }
+                playerSession.update(x -> x.withCurrentSection(SectionId.book1(310)));
+            }
+            case FOLLOW_SOUTH -> {
+                if (!hasDiceOptionBeenSelected()) {
+                    throw new FabledBusinessException("The Dice Option was not clicked");
+                }
+                playerSession.update(x -> x.withCurrentSection(SectionId.book1(10)));
+            }
+            case GO_WEST -> {
+                if (!hasDiceOptionBeenSelected()) {
+                    throw new FabledBusinessException("The Dice Option was not clicked");
+                }
+                playerSession.update(x -> x.withCurrentSection(SectionId.book1(558)));
+            }
+            case GO_EAST -> {
+                if (!hasDiceOptionBeenSelected()) {
+                    throw new FabledBusinessException("The Dice Option was not clicked");
+                }
+                playerSession.update(x -> x.withCurrentSection(SectionId.book1(278)));
+            }
+        }
+    }
+
+    @Override
+    public SectionDto.SectionTicks getTicks() {
+        return SectionHandler.super.getTicks();
+    }
+
+    private void addPlayerClickId(int clickId) {
+        playerSession.update(x -> {
```

Comment:
nice idea (we could even move this outside of the section to make this feature of "passed clicks" available for free for everyone as part of the overall game mechanic).
one thing to consider: when re-visiting the same section. the list should be cleared...currently I think it's not (or I missed it).

Reply:
I can do this in another PR for cleaning up section 82